deploy/docker-compose.yml

```diff
@@ -36,6 +36,7 @@ services:
       - GITHUB_CLIENT_SECRET
       - GITHUB_TOKEN
       - WEBHOOK_URL
+      - ECR_ENDPOINT
```

Comment:
This is not really necessary because we don't consume this env var within the container (only env vars that the hub running inside of docker needs to consume go here)--rather, the only consumer of this env var is the build.js script running in travis.

Reply:
This is removed now.